config/locales/sv.yml

```diff
@@ -1179,6 +1189,8 @@ sv:
         new_password: Nytt lösenord
         info_header: Redigera namn och e-post
         password_confirmation: *password_confirmation
+
+    shared:
```

Comment:
`member_photo:pic_header` should go under `devise:registrations:edit`
(The organization is about the action, not the partial file name)